cmd/validator/validator.go

```diff
@@ -113,30 +130,20 @@ func getFlags() (validatorConfig, error) {
 		"depth":              "CFV_DEPTH",
 		"exclude-dirs":       "CFV_EXCLUDE_DIRS",
 		"exclude-file-types": "CFV_EXCLUDE_FILE_TYPES",
+		"file-types":         "CFV_FILE_TYPES",
 		"output":             "CFV_OUTPUT",
 		"reporter":           "CFV_REPORTER",
 		"groupby":            "CFV_GROUPBY",
 		"quiet":              "CFV_QUIET",
 	}
 
-	for flagName, envVar := range flagsEnvMap {
-		if err := setFlagFromEnvIfNotSet(flagName, envVar); err != nil {
-			return validatorConfig{}, err
-		}
+	if err := setFlagFromEnvIfNotSet(flagsEnvMap); err != nil {
+		return validatorConfig{}, err
 	}
 
 	flag.Parse()
 
-	searchPaths := make([]string, 0)
-
-	// If search path arg is empty, set it to the cwd
-	// if not, set it to the arg. Supports n number of
-	// paths
-	if flag.NArg() == 0 {
-		searchPaths = append(searchPaths, ".")
-	} else {
-		searchPaths = append(searchPaths, flag.Args()...)
-	}
+	searchPaths := buildSearchPath(flag.NArg())
```

Comment:
I don't see a need to pass this I would use this.

```suggestion
	searchPaths := buildSearchPath()
```

flag is "global" and you are already accessing flag.Args in the method

So another solution would be to path

```suggestion
	searchPaths := buildSearchPath(flag.Args())
```

Then use len in the method